tests/test_layers.py

```diff
@@ -525,7 +526,7 @@ def test_gaussian_pooling_train():
                           learning_rate=1e-3,
                           normalize=(True, True))
     model.layers[0].build(xtrain.shape)
-    assert len(model.layers[0].trainable_weights) == 0
+    assert not any(model.layers[0].trainable_weights)
```

Comment:
nitpick but i often prefer len()==0 because if the inside is [0] then we sometimes get messed up on not(any([0]))

Reply:
Yeah forgot about that edge case